webview/package.json

```diff
@@ -7,6 +7,7 @@
     "lint:prettier": "prettier -c '**/*.{js,json,jsx,md,scss,ts,tsx,yaml,yml}'",
     "lint:eslint": "eslint --cache '**/*.{js,ts,jsx,tsx}'",
     "lint:tsc": "tsc --noEmit",
+    "lint:webpack": "webpack --mode development",
```

Comment:
[Q] Do we need `lint:tsc` if we have this? Could we change to `lint:compile` for all sub-projects? Leave `extension` as is but change the two sub-projects bundled by webpack to use this?

Reply:
That's a good point, thanks! I'll add lint:compile to the two webpacked projects and remove lint:tsc from them